_data/students.yml

```diff
@@ -1,8 +1,14 @@
+- name: Amriyono Maju Ompusunggu
+  country: Indonesia
+  github: amryarios
+  image: me.jpg
```

Comment:
It's better if you could rename `me.jpg` with `yourName.jpg` and for the Image too.